doc/source/serve/http-guide.md

```diff
@@ -40,6 +40,12 @@ Often for ML models, you just need the API to accept a `numpy` array. You can us
 Serve provides a library of HTTP adapters to help you avoid boilerplate code. The [later section](serve-http-adapters) dives deeper into how these works.
 ```
 
+Please note that different metrics are collected when calling Deployments
```

Comment:
Can we use the note syntax (and update the wording) here as well:

```markdown
:::{note}
Different metrics are collected when Deployments are called
via Python `ServeHandles` and when they are called via HTTP.

See the [Ray Serve monitoring documentation](serve-monitoring) for more details.
:::
```

Reply:
Updated